packages/tests/src.ts/test-providers.spec.ts

```diff
@@ -1063,15 +1060,105 @@ describe("Test Hedera Provider", function () {
         assert.strictEqual(true, balance.gte(0));
     }).timeout(timeout);
 
-    it("Gets txn record", async function (){
+    //TODO add formatter tests ->
+
+    describe("Sign & Send Transacton, Wait for receipt", function () {
+
+        // let sendTransactionResponse: ethers.providers.TransactionResponse;
+        let signedTx: string | Promise<string>;
+        beforeEach(async () => {
+            const privateKey = PrivateKey.fromString(hederaTestnetOperableAccount.operator.privateKey);
+            // 1. Sign TX -> `sign-transaction.ts`
+            const txID = TransactionId.generate(hederaTestnetOperableAccount.operator.accountId);
+            const tx = await new ContractCreateTransaction()
+                .setContractMemo("memo")
+                .setGas(100000)
+                .setBytecodeFileId("0.0.26562254")
+                .setNodeAccountIds([new AccountId(0,0,3)])
+                .setConstructorParameters(new ContractFunctionParameters().addUint256(100))
+                .setTransactionId(txID)
+                .freeze()
+                .sign(privateKey);
+            const txBytes = tx.toBytes();
+            signedTx = ethers.utils.hexlify(txBytes);
+            // const provider = ethers.providers.getDefaultProvider('testnet');
+            // sendTransactionResponse = await provider.sendTransaction(signedTx);
+        });
+
+        it.only("Schould populate transaction receipt", async function (){
+            const sendTransactionResponse = await provider.sendTransaction(await signedTx);
+            const receipt = await sendTransactionResponse.wait();
+            // assert.strict(receipt.logs.length > 0);
+            assert.strictEqual(receipt.to, null);
+            assert.strictEqual(receipt.contractAddress, '0x'+sendTransactionResponse.customData.contractId);
+            assert.strictEqual(receipt.from, getAddressFromAccount(hederaTestnetOperableAccount.operator.accountId));
+            assert.strictEqual(receipt.transactionHash, sendTransactionResponse.hash);
+        }).timeout(timeout);
+
+        it.only("Schould populate transaction receipt with timeout", async function (){
```

Comment:
typo + `only` usage - we need all the tests in the CI pipeline

Reply:
leftovers, corrected now